Cubical/Algebra/Group/QuotientGroup.agda

```diff
@@ -0,0 +1,105 @@
+{-
+
+This file contains quotient groups
+
+-}
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+module Cubical.Algebra.Group.QuotientGroup where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Structure
+open import Cubical.Foundations.Powerset
+open import Cubical.Foundations.GroupoidLaws hiding (assoc)
+open import Cubical.Data.Sigma
+open import Cubical.HITs.SetQuotients.Base renaming (_/_ to _/s_)
+open import Cubical.HITs.SetQuotients.Properties
+open import Cubical.Relation.Binary.Base
+
+open import Cubical.Algebra.Group.Base
+open import Cubical.Algebra.Group.Properties
+open import Cubical.Algebra.Group.Morphisms
+open import Cubical.Algebra.Group.MorphismProperties
+open import Cubical.Algebra.Group.Subgroup
+
+private
+  variable
+    ℓ : Level
+
+module _ (G' : Group {ℓ}) (H' : Subgroup G') (Hnormal : isNormal H') where
+
+  open BinaryRelation
+  open isSubgroup (snd H')
+  open GroupStr (snd G')
+  open GroupTheory G'
+  private
+    G = ⟨ G' ⟩
+
+  _~_ : G → G → Type ℓ
+  x ~ y = x · inv y ∈ ⟪ H' ⟫
+
+  isRefl~ : isRefl _~_
+  isRefl~ x = subst-∈ ⟪ H' ⟫ (sym (invr x)) id-closed
+
+  G/H : Type ℓ
+  G/H = G /s _~_
+
+  1/H : G/H
+  1/H = [ 1g ]
+
+  _·/H_ : G/H → G/H → G/H
+  x ·/H y = setQuotBinOp isRefl~ _·_ rem x y
```

Comment:
@felixwellen  @mzeuner We should use this function (or the version for commutative operations) in QuotientRing. It should make the definition a bit slicker